src/components/PipelineRunListView/PipelineRunListHeader.ts

```diff
@@ -2,6 +2,7 @@ export const pipelineRunTableColumnClasses = {
   name: 'pf-m-width-20 wrap-column',
   status: 'pf-m-width-10',
   started: 'pf-m-width-20',
+  vulnerabilities: 'pf-m-hidden pf-m-visible-on-xl pf-m-width-10',
```

Comment:
Started column takes more width than it is needed and vulnerabilities column text wraps due it not enough space.

<img width="1485" alt="Screenshot 2023-03-15 at 11 32 22 AM" src="https://user-images.githubusercontent.com/9964343/225220889-88099726-61b0-498b-9b7b-3fb4e8c9a994.png">

 we can evenly distribute the widths between these columns, so the column data wont wrap.
```suggestion
  started: 'pf-m-width-15',
  vulnerabilities: 'pf-m-hidden pf-m-visible-on-xl pf-m-width-15',
```